docs/docs/content_recommend.md

```diff
@@ -102,11 +108,36 @@ Recommend controls any parameters necessary for making a content suggestions to
   <tr>
     <td>raw</td>
     <td>string</td>
-    <td>raw FilterQL formatted query (ex. "FILTER AND (global.mobile_experience > 0) FROM content")*</td>
+    <td>raw FilterQL formatted query ex. <code>FILTER AND (global.mobile_experience > 0) FROM content</code>*</td>
+  </tr>
+</table>
+
+<table>
+  <thead>
+    <tr>
+      <td colspan="3" align="center"><code>ast</code> object</td>
+    </tr>
+    <tr>
+      <th>Key</th>
+      <th>Type</th>
+      <th>Behavior</th>
+    </tr>
+  </thead>
+  
+  <tr>
+    <td>args</td>
+    <td>array</td>
+    <td>list of arguments to be evaluated with the operator, these arguments may be identifiers, literals, or further expressions*</td>
```

Comment:
"operator. These" to prevent a run-on sentence. Other than that nit-pick these are successfully vague yet informative 👍 

Hopefully soon we will have a destination page on the AST spec.